File: cash.py
from cs50 import get_float
from math import floor

def Obtain_Change():
    while True:
        change = get_float("Change owed: ")
        if (change > 0):
            cents = change * 100
            return cents

x = Obtain_Change()

Num_of_coins = 0

while x > 0:
    if x >= 25:
        x -= 25
        Num_of_coins += 1
    elif x >= 10:
        x -= 10
        Num_of_coins += 1
    elif x >= 5:
        x -= 5
        Num_of_coins += 1
    elif x >= 1:
        x -= 1
        Num_of_coins += 1

print(Num_of_coins)

File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int Obtain_Change(string prompt);

int main(void)
{
    int x = Obtain_Change("Cents");
    int Num_of_coins = 0;

    // Get the amount of coins
    Num_of_coins += x / 25;
    x %= 25;

    Num_of_coins += x / 10;
    x %= 10;

    Num_of_coins += x / 5;
    x %= 5;

    Num_of_coins += x / 1;
    x %= 1;

    printf("%i\n", Num_of_coins);
}

// Get positive integer
int Obtain_Change(string prompt)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);
    int cents = round(change * 100);
    return cents;
}
